taichi/program/context.h

```diff
@@ -4,8 +4,8 @@
 #include "taichi/inc/constants.h"
 
 #if defined(TI_RUNTIME_HOST)
```

Comment:
Can we use a reversed logic, i.e. TI_RUNTIME_DEVICE? Since it's in program/ not runtime/

Reply:
Thanks for proposing, but I actually think the current implementation is fine.